src/main/java/com/olgaboiar/mint/Logger.java

```diff
@@ -0,0 +1,23 @@
+package com.olgaboiar.mint;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+
+public class Logger {
+
+    public void logToConsole(String message) {
+        System.out.println(message);
+    }
+    
+    public void logToFile(String message, String fileName) throws IOException {
```

Comment:
Can we allow the user to set the filename once instead of each time they call the log method?

Reply:
refactored in https://github.com/olgaboiar/MintHTTPServer/pull/6/commits/c76f01df3c42aa7c34ef97cdb04a5b8319149825